addons/udes_stock/models/stock_picking.py

```diff
@@ -10,6 +10,116 @@ class StockPicking(models.Model):
         "Sequence", default=0, help="Used to order the 'All Operations' kanban view"
     )
 
+    # Related pickings computed fields
+
+    u_first_picking_ids = fields.One2many(
+        "stock.picking",
+        string="First Pickings",
+        compute="_compute_first_picking_ids",
+        help="First pickings in the chain",
+    )
+    u_prev_picking_ids = fields.One2many(
+        "stock.picking",
+        string="Previous Pickings",
+        compute="_compute_related_picking_ids",
+        help="Previous pickings",
+    )
+    u_next_picking_ids = fields.One2many(
+        "stock.picking",
+        string="Next Pickings",
+        compute="_compute_related_picking_ids",
+        help="Next pickings",
+    )
+    u_created_backorder_ids = fields.One2many(
+        "stock.picking",
+        string="Created Back Orders",
```

Comment:
Do we want to change this to `Backorders` as well?

Reply:
Not sure, both variants are used throughout UDES.

Will ask in the group and see what the consensus is. Can then update the other instances of Backorders/Back Orders so that they're consistent. :)

Reply:
Have now replaced instances of 'Back order' with 'Backorder'.